src/Exception/ParsingException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\Exception;
+
+use Exception;
+use Rector\Core\ValueObject\Error\SystemError;
+
+final class ParsingException extends Exception
+{
+    public function __construct(public readonly SystemError $systemError)
```

Comment:
I'll make this property private in next rebase+push after dependency PRs are merge.
It's a leftover from a previous test